src/mugs.js

```diff
@@ -719,6 +719,15 @@ define([
             data.instances = _.extend(data.instances || {},
                                       mug.form.parseInstanceRefs(value));
         }
+        try {
+            if (value) {
+                value = mug.form.xpath.parse(value.toString()).toHashtag();
+            }
+        } catch (err) {
+            if (_.isString(value) && !value.startsWith('#invalid/')) {
+                value = '#invalid/xpath ' + value;
```

Comment:
how is this deserialized? is the `#invalid/xpath ` prefix stripped before displaying to the user? where does that happen (it doesn't appear to be handled below in `deserializeXPath`, which seems not great because serialize/deserialize are supposed to be symmetrical.